modules/Lending.py

```diff
@@ -29,7 +33,6 @@
 coin_cfg = {}
 dry_run = 0
 transferable_currencies = []
-currencies_to_analyse = []
```

Comment:
Is this related to the change?

Reply:
I dont know exactly why my file had `exchange, analysis_method, currencies_to_analyse` taken out and replaced with `exchange, analysis_method`, maybe I had a slightly older file? Same with `if Analysis and cur in currencies_to_analyse:` and `if Analysis:` 

Sorry. Also I just realized that in my default.cfg I had `max_daily_rate` uncommented... and when I commented it to set it to the infinite default value the bot ceased to work.